Compactable/separate

```diff
@@ -0,0 +1,49 @@
+{-
+
+mapEither maps a function to Either over the `f` and `separate`s `Left`s from `Right`s.
```

Comment:
This says `mapEither`, but the file is `separate`. I would change this comment to say something like “This expression maps …” to make it less likely to get out of sync.